posts/views.py

```diff
@@ -14,11 +16,14 @@ def post(self, request):
         try:
             title = request.data['title']
             content = request.data['content']
-            author = request.author
-            post = Post.objects.create(title=title, content=content, user_id=author)
+            user_id = request.data['user_id']
+            author = UserProfile.objects.get(user_id=user_id)
```

Comment:
```suggestion
            author_id = request.data['author']
            author = UserProfile.objects.get(user_id=author_id)
```

author 로 받으면 위 코드처럼 되겠네요

Reply:
감사합니다 ㅎㅎ!